database/cvsCreater.js

```diff
@@ -2,10 +2,29 @@
 const fs = require('fs');
 const json2csv = require('json2csv');
 const fileSize = 10000000;
-const chunkSize = fileSize / 10;
+const chunkSize = fileSize / 100; //speed goes up the smaller the chunk size, but speed seems to slow down after 100-1000 (takes about ~15s)
```

Comment:
curious if you have messed with "node --max-old-space-size" property at all?  You can allocate more RAM to the seed script when you run it.